frontend/packages/console-shared/src/components/status/fixtures/status.fixture.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { Status, PopoverStatus } from '../status';
+
+export default [
```

Comment:
@jtomasek If I understand correctly, this is the [Cosmos](https://github.com/react-cosmos/react-cosmos) fixture format, right?

@christianvogt This could be another use case for `console-scripts` package - add Cosmos dependency & configuration and use it in packages that wish to leverage this tool.